### DISCUSSION TITLE
[Cursed Discussion Title]

### DISCUSSION THREAD

**Discussion by Nadelio** [Nadelio/SOTF#5] [Announcements]
##  Help.
  We’re using Discussions as a way to help me and other people get better at coding in Python:
  * Please for the love of God make my code better
  * Give me ideas for new features
  * Teach me ways I can improve my code and expand my Python knowledge
  * Discuss with one another about the game
  * Also seriously, keep the discussions relevant to THIS REPO, I'm looking at you @Sheriff-Nevarez  👀 

  [Insert generic introduction here]

**Comment by Sheriff-Nevarez** [comment]
[Cursed Discussion Text]

[Insert goofy goodbye here]
